spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```
@@ -1125,6 +1126,7 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
         } else {
           if (matchAll) new Exception(s"module $name cannot find corresponding weight bias")
         }
+      case _=>
```

you should throw an exception here, or mismatch will do nothing.